src/sso/properties.ts

```diff
@@ -1,10 +1,12 @@
+import { WebViewErrorEvent } from "react-native-webview/lib/WebViewTypes"
```

Comment:
Question: can we import this from `react-native-webview` instead of `react-native-webview/lib/WebViewTypes`?